test/xpu_api/random/conformance_tests/minstd_rand_rand0_test.pass.cpp

```diff
@@ -50,7 +50,7 @@ int main() {
         }
     };
 
-    sycl::queue queue(sycl::default_selector{}, exception_handler);
+    sycl::queue queue(exception_handler);
```

Comment:
```suggestion
    sycl::queue queue(TestUtils::default_selector, exception_handler);
```

This is not directly related to this PR, but we need to use  `TestUtils::default_selector` in order to resolve ambiguity when running tests on FPGA hardware (we have both FPGA hardware end emulator in the environment then).

That can only be done within our tests, because device selection environment variables (see [EnvironmentVariables](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md#oneapi_device_selector) for details) do not differ FPGA and FPGA emulator.